schema/manifest.schema.json

```diff
@@ -104,16 +104,41 @@
       },
       "default": []
     },
+
     "proxy": {
-      "type": "object",
+      "type": ["object", "array"],
       "default": {},
-      "maxProperties": 4,
-      "patternProperties": {
-        ".*": {
-          "type": "string",
-          "format": "uri-reference"
+      "anyOf": [
+        {
+          "type": "object",
+          "maxProperties": 4,
```

Comment:
Out of curiosity, why are we setting max properties here? Looks like we're setting `additionalProperties` to false below